### PR TITLE
revert: replace just the first `/` with a `~` in ids

### PR DESCRIPTION
**Pull request by vladfrangu**

### CODE REVIEW

File: src/base/api_client.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@ export abstract class ApiClient {
     }
 
     protected _toSafeId(id: string): string {
-        return id.replace(/\//g, '~');
+        return id.replace('/', '~');
```
Comment (szmarczak) [inline src/base/api_client.ts:73]: Can you please add a comment why's that? So we don't go into the same pitfall again :)

Comment (vladfrangu) [reply src/base/api_client.ts:73]: Would a link to the slack thread be enough? Should I explain it instead or?

Comment (szmarczak) [reply src/base/api_client.ts:73]: A comment would be best. Something like `The id looks like 'username/project' so we need to replace the slash only once` or so.

Comment (vladfrangu) [reply src/base/api_client.ts:73]: Actually, probably explaining it would be the best so other contributors can see it at a glance

**Review by B4nan — APPROVED** [review]

**Review by szmarczak — APPROVED** [review]

**Pull request merged**